libqtile/core/manager.py

```diff
@@ -738,6 +738,16 @@ def process_button_click(self, button_code: int, modmask: int, x: int, y: int) -
                         continue
                 else:
                     val = (0, 0)
+
+                if m.warp_pointer:
+                    i = m.warp_pointer
+                    status, warp_to = self.server.call((i.selectors, i.name, i.args, i.kwargs))
+                    if status in (interface.ERROR, interface.EXCEPTION):
+                        logger.error("Mouse command error %s: %s", i.name, val)
+                        continue
+                    x, y = warp_to[0], warp_to[1]
+                    self.core.warp_pointer(x, y)
```

Comment:
```suggestion
                    self.core.warp_pointer(*warp_pointer)
```

Reply:
This should be

```
self.core.warp_pointer(*warp_to)
``` 

:)

Reply:
Oops. Yes!

Reply:
We need to overshadow `x, y` that come in as method parameters so then `self._drag = (x, y, val[0], val[1], m.commands)` a few lines later has the correct values

Reply:
Cool. That makes sense.